pkg/services/serviceaccounts/api/api.go

```diff
@@ -218,7 +218,14 @@ func (api *ServiceAccountsAPI) SearchOrgServiceAccountsWithPaging(c *models.ReqC
 	if page < 1 {
 		page = 1
 	}
-	serviceAccountSearch, err := api.store.SearchOrgServiceAccounts(ctx, c.OrgId, c.Query("query"), page, perPage, c.SignedInUser)
+	expiredTokensFilterQuery := c.Query("expiredTokens")
+	// its okay that it fails, it is only filtering that might be weird, but to safe quard against any weird incoming query param
+	onlyWithExpiredTokens, _ := strconv.ParseBool(expiredTokensFilterQuery)
```

Comment:
```suggestion
	// its okay that it fails, it is only filtering that might be weird, but to safe quard against any weird incoming query param
	onlyWithExpiredTokens := c.QueryBool("expiredTokens")
```

Reply:
I was trying to cover for parsing that failed to create a boolean, but I guess we should expect correct parameters from the frontend query

Reply:
what happens to QueryBool, if it fails to see a boolean or anything in particular? 🤔

Reply:
it defaults to false I hope >D

Reply:
I'm not sure I follow, it seems to me `QueryBool` does internally exactly the same as you are doing:
```go
// QueryBool returns query result in bool type.
func (ctx *Context) QueryBool(name string) bool {
	v, _ := strconv.ParseBool(ctx.Query(name))
	return v
}
```

Reply:
```
// ParseBool returns the boolean value represented by the string.
// It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False.
// Any other value returns an error.
func ParseBool(str string) (bool, error) {
	switch str {
	case "1", "t", "T", "true", "TRUE", "True":
		return true, nil
	case "0", "f", "F", "false", "FALSE", "False":
		return false, nil
	}
	return false, syntaxError("ParseBool", str)
}
```
thats great